pkg/util/docker/dockerfile/dockerfile.go

```diff
@@ -0,0 +1,55 @@
+package dockerfile
```

Comment:
> just wondering why it's not in util/docker?

@bparees I went for the longer `util/docker/dockerfile` to keep it tidy and focused. We already have a `dockerutil.go` that could well go to `util/docker` or some other subpackage, while these Dockerfile utilities are independent of the other things in `dockerutil.go`, that defined `type DockerClient`, `func pushImage()`, among other things...

Also, when importing, it reads better:
```go
import "github.com/openshift/origin/pkg/util/docker/dockerfile"

dockerfile.InsertInstructions(node, 0, "")
```